develop/tutorials/articles/230-web-experience-management/01-developing-fragments/02-fragment-specific-tags.markdown

```diff
@@ -32,15 +32,6 @@ image before publishing---including those from Documents and Media.
 `rich-text`: Similar to the text field, but provides an WYSIWYG editor for
```

Comment:
Provides rich text formatting capabilities to the page author, such as bold, italics, underline, links and pre-defined styles.